hana/monitoring.sls

```diff
@@ -1,34 +1,32 @@
 {%- from "hana/map.jinja" import hana with context -%}
 {%- from 'hana/macros/get_hana_client_path.sls' import get_hana_client_path with context %}
 
-{% set pydbapi_output_dir = '/tmp/pydbapi' %}
-
-{% for node in hana.nodes if node.host == grains['host'] %}
-{%- set hana_client_path = get_hana_client_path(hana, node) %}
+{%- set pydbapi_output_dir = '/tmp/pydbapi' %}
+# first element in nodes has to be the primary
+{%- set node0 = hana.nodes[0] %}
```

Comment:
This looks a potential source of errors.
We don't know if we will always have 2 nodes in the pillar file. Or even if we will have more nodes (potentially scale out, as this PR is working on)